packages/config/config/devices/0x027a/zen26_2.0-2.01.json

```diff
@@ -0,0 +1,303 @@
+// Zooz ZEN26
+// Z-Wave Plus S2 ON/OFF Switch
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN26",
+	"description": "Z-Wave Plus S2 ON/OFF Switch",
+	"devices": [
+		{
+			"productType": "0xa000",
+			"productId": "0xa001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "2.01"
```

Comment:
Please use the minimum number of digits necessary for the minor number. This should be 2.1 (even though Zooz probably lists 2.01). Please carry throughout.